Framework/PythonInterface/mantid/api/src/Exports/SpectrumInfo.cpp

```diff
@@ -14,5 +14,15 @@ void export_SpectrumInfo() {
            "masked.")
       .def("hasDetectors", &SpectrumInfo::hasDetectors, (arg("self")),
            "Returns true if the spectrum is associated with detectors in the "
-           "instrument.");
+           "instrument.")
+      .def("position", &SpectrumInfo::position, (arg("self"), arg("index")),
+           "Returns the position of the detector(s) associated with the "
+           "spectrum.")
+      .def("l2", &SpectrumInfo::l2, (arg("self"), arg("index")),
+           "Returns the distance from the sample of the detector(s) "
+           "associated with the spectrum.")
+      .def("phi", &SpectrumInfo::phi, (arg("self"), arg("index")),
+           "Returns phi angle of the detector(s) associated with the spectrum.")
+      .def("twoTheta", &SpectrumInfo::twoTheta, (arg("self"), arg("index")),
+           "Returns twoTheta of the detector(s) associated with the spectrum.");
```

Comment:
We are considering renaming this method, since there is some risk of confusing with the `theta` in sperical coordinates. I this did not expose it to Python, since it may be removed, and it would not make sense to deprecate functions even before the interface is complete.